### PR TITLE
Define contract for turbine response and capture

### PR DESCRIPTION
**Pull request by jmar910**
## Description of change

CLI now expects a specific format when receiving responses from turbine-js. This allows us to only capture what we care about and leave the rest of the output alone. 

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [x]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

## Additional references

See this thread https://meroxa.slack.com/archives/C02L14X6TA9/p1649879600763529

## Blockers
+ [ ] Merge https://github.com/meroxa/turbine-js/pull/58 at the same time

### CODE REVIEW

File: cmd/meroxa/turbine_cli/javascript/deploy.go [hunk 1]
```diff
@@ -13,15 +14,25 @@ import (
...
 	if err != nil {
-		err := fmt.Errorf(
-			"unable to determine if the Meroxa Application at %s has a Process; %s",
-			path,
-			string(output))
-		return false, err
+		return processError(err, path, output)
+	}
+
+	r, _ := regexp.Compile("\nturbine-response: (true|false)\n")
```
Comment (janelletavares) [inline cmd/meroxa/turbine_cli/javascript/deploy.go:32]:
```suggestion
	r := regexp.MustCompile("\nturbine-response: (true|false)\n")
```
nit

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I took a spin on this locally and noticed that app initialization and runs work great with regards to output and error handling, but something seems amiss with deploying apps -- is this related to this change?

```bash
$ ./meroxa apps deploy --path ../demos/myjsapp
Checking for uncommitted changes...
	✔ No uncommitted changes!
Validating branch...
	✔ Deployment allowed from master branch!
Preparing application "myjsapp" (javascript) for deployment...
	✔ Application built!
	
Error: strconv.ParseBool: parsing "turbine-response: true": invalid syntax
```

The recording includes
- a failed deployment of an app instance to the same namespace `myjsapp`
- another subsequent redeployment to the `myjsapp` after deleting the previous app that also fails

democlioutput

**Comment by jmar910** [comment]
@jayjayjpg just to confirm, did you pull the `james/update-response-format` branch from https://github.com/meroxa/turbine-js/pull/58 when you tested this locally? I forgot to add it in this PR template, but these two PRs have to be merged in lockstep

**Comment by jayjayjpg** [comment]
@jmar910 Yes exactly, but let me double check and record just to make sure, also happy to huddle if it's helpful!

**Review by jayjayjpg — APPROVED** [review]
After @jmar910 paired with me we noticed that I failed to pull the latest `master` before testing and this change is already working as expected -- awesome work!

democlioutput3

**Pull request merged**